tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/proton/ProtonTest.java

```diff
@@ -211,6 +213,25 @@ public void testBrokerContainerId() throws Exception {
    }
 
    @Test
+   public void testBrokerConnectionProperties() throws Exception {
+      if (protocol != 0 && protocol != 3) return; // Only run this test for AMQP protocol
```

Comment:
Yes.  It'd be better to use an AssumeTrue or similar.   We do this in many other areas of the test class, so perhaps we can update the to use the junit assertions in a separate commit.